terraform/terraform.tfvars

```diff
@@ -15,3 +15,5 @@
 gcp_project_id = "<project_id_here>"
 
 memorystore = false
+
+redis_enterprise = false
```

Comment:
**Rationale for change:** I've changed the `redis-enterprise` variable to `redis_enterprise` just to be consistent with the casing of other variables (e.g., `gcp_project_id`). Not sure if there's a standard casing that Terraform officially recommends — but better to be consistent.